src/umpire/strategy/QuickPool.cpp

```diff
@@ -262,9 +265,11 @@ std::size_t QuickPool::getActualSize() const noexcept
   return m_actual_bytes;
 }
 
+// TODO: Keep old behaviour with m_aligned_bytes or remove and use definition
+// from AllocationStrategy which returns m_current_size?
```

Comment:
This also needs addressing, same as https://github.com/LLNL/Umpire/pull/908/files#r1749860382.

Reply:
Same comment as above.

Reply:
Removed in https://github.com/LLNL/Umpire/pull/908/commits/58a685af3f7ab6e3002995cf83e6b5c25e26af11.